packages/grafana-schema/src/common/data_source.cue

```diff
@@ -0,0 +1,68 @@
+package common
+
+// TODO docs
+DataSourceJsonData: {
+  authType?: string
+  defaultRegion?: string
+  profile?: string
+  manageAlerts?: bool
+  alertmanagerUid?: string
+} @cuetsy(kind="interface")
+
+// Frontend settings model that is passed to Datasource constructor. This differs a bit from the model above
+// as this data model is available to every user who has access to a data source (Viewers+).  This is loaded
+// in bootData (on page load), or from: /api/frontend/settings
+DataSourceInstanceSettings: {
+	id: int64
```

Comment:
The formatting looks a bit off on some of the fields, but it looks fine in my EDI, so not sure what's the issue here.